### PR TITLE
Added publishing the remote library state on comic update [#1327]

### LINKED ISSUE #1327: Library statistics updates are published when a comic is changed.
As an application, when any comic changes state in the library, a library update is sent to all browsers.

## Dependencies
1. #1284

### PR DESCRIPTION
**Pull request by mcpierce**
**NOTE:** This PR is based on #1283 and that PR needs to be merged first.

Closes #1327 

## Status
READY

## Does this PR contain migrations?
NO

## Before You Submit Your PR:
- [ ] Have you announced your PR on the comixed-dev mailing list?
- [X] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactors existing code (code changes for efficiency or maintainability)
- [ ] Security fix (be sure to include the CVE in the commit message) 
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

**Comment by BRUCELLA2** [comment]
can you rebase this branch? It will remove the conflicts now and it will make the review easier :)

**Comment by mcpierce** [comment]
> can you rebase this branch? It will remove the conflicts now and it will make the review easier :)

Rebased and pushed.

File: comixed-services/src/test/java/org/comixedproject/service/library/RemoteLibraryStateServiceTest.java [hunk 1]
```diff
@@ -23,24 +23,32 @@
...
 
 @RunWith(MockitoJUnitRunner.class)
 public class RemoteLibraryStateServiceTest {
   private static final long TEST_COMIC_COUNT = Math.abs(RandomUtils.nextLong());
   private static final long TEST_DELETED_COMIC_COUNT = Math.abs(RandomUtils.nextLong());
 
   @InjectMocks private RemoteLibraryStateService service;
+  @Mock private ComicStateHandler comicStateHandler;
   @Mock private ComicBookService comicBookService;
+  @Mock private RemoteLibraryStateService remoteLibraryStateService;
```
Comment (BRUCELLA2) [inline comixed-services/src/test/java/org/comixedproject/service/library/RemoteLibraryStateServiceTest.java]: To be deleted (not used)

**Review by BRUCELLA2 — CHANGES_REQUESTED** [review]

File: comixed-services/src/test/java/org/comixedproject/service/library/RemoteLibraryStateServiceTest.java (hunk 1 above)
Comment (mcpierce) [reply comixed-services/src/test/java/org/comixedproject/service/library/RemoteLibraryStateServiceTest.java]: Removed it.

**Review by BRUCELLA2 — APPROVED** [review]

**Pull request merged**